utils.py

```diff
@@ -61,16 +62,17 @@ def slope_sym(traj, x, y):
     # df/dx / df/dy -> dy/dx
     # TODO: does this hold for things that can't be written y = f(x)?
     # tan(-x) = -tan(x)
-    slope = -1 * diff(traj, x) / diff(traj, y)
-    return slope
+    # slope = -1 * diff(traj, x) / diff(traj, y)
+    return -1 * diff(traj, x), diff(traj, y)
 
 
 def find_transitions(trajectory, angles, x, y) -> dict:
     # NOTE: trajectory is an *expression*, not equation
     transitions = {}
     for angle in angles:
         # Compute slope symbolically
-        slope = slope_sym(trajectory, x, y)
+        df_dy, df_dx = slope_sym(trajectory, x, y)
+        slope = df_dy/df_dx
         # TODO(nishant): should this be atan2 here?
         # solve(Eq(atan2(dfdy, dfdx), angle))??
         soln = solve(Eq(slope, tan(angle)))
```

Comment:
did you get a chance to try using `atan2` here?

Reply:
Oh I forgot to change that part, I'll try atan2 there